broker/config/settings.yml

```diff
@@ -187,6 +187,12 @@ defaults: &defaults
       key: <%= certificate('broker.key') %>
       cert: <%= certificate('broker.crt') %>
 
+  apiserver:
+    ip: '127.0.0.1'
+    port: 9443
+    encryption:
+      password: 'ABCDEFGHABCDEFGHABCDEFGHABCDEFGH'
+      initialization_vector: 'ABCDEFGHABCDEFGH'
```

Comment:
suggestion: while putting in actual config generate using random byte generator as IV is recommended as cryptographically random and unpredictable. something like following
`crypto.randomBytes(64).toString('hex')`

Reply:
settings.yml is just an example.

Reply:
+1 - this could be injected into from manifet.erb from product component. Agree above is just sample.